go/cognito-httpapi/README.md

```diff
@@ -0,0 +1,16 @@
+# Amazon Cognito to Amazon API Gateway HTTP APIs (JWT)
```

Comment:
Please expand this. 

There's very little content here. We generally like larger, end to end covering from "want to do" through to "end up like this".

Tell me what this example does. Tell me how to deploy it. Tell me how I know it worked.